### PR TITLE
Add first book chapter summaries

### LINKED ISSUE #741: [Ticket]: Write chapter summaries for the Harry Potter Books
### Project

API (api.potterdb.com)

### Describe the ticket you want to create

Our API is currently missing an essential component - chapter summaries for the Harry Potter books. 
I am aware this is a huge task and big effort that is why I decided to split it into multiple parts.

### Get started
- Choose one of the books from Harry Potter you would like to write chapter summaries for and leave a notice below. 
- Craft insightful chapter summaries that capture the essence of each chapter. The data for the books is located in [backend/db/data/books](https://github.com/danielschuster-muc/potter-db/tree/master/backend/db/data/books). For inspiration look at this [discussion](https://github.com/danielschuster-muc/potter-db/discussions/170). 
- Submit your pull request and mention the book's title. 

You can contribute chapter summaries for multiple books, but please work on one book (with all chapters) per pull request. If you want you can also work together with other on the same branch. 

If you have questions or need guidance on formatting or content, feel free to engage in the linked GitHub discussion or ask for assistance on this issue.

### Acceptance Criteria

- [ ] Choose a book and mention it below.
- [ ] Write concise, informative summaries for each chapter of your choosen book. 

### Additional Details

https://github.com/danielschuster-muc/potter-db/discussions/170

### Screenshots

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by kat-kan**
## Summary

I added first book summaries in 1.json file

## Related Issues
#741

## Checklist

- [x] This pull request has a meaningful title and description.
- [x] I have read and followed the [Contributing guidelines](https://github.com/danielschuster-muc/potter-db/blob/master/CONTRIBUTING.md).
- [x] My code follows the conventions and coding style of this project.
- [x] All tests pass and I have added tests for my changes  - N/A 
- [x] I have updated necessary documentation N/A 

# Screenshots (if applicable)

### CODE REVIEW

**Comment by kat-kan** [comment]
@danielschuster-muc can you please review?

File: backend/db/data/books/1.json [hunk 1]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "The Mirror of Erised",
-      "order": "12"
+      "summary": "Harry discovers a mysterious mirror in an abandoned classroom at Hogwarts. When he looks into the mirror, he sees himself with his deceased parents, James and Lily Potter, who he never had a chance to know. The mirror shows the deepest desires of a person's heart, and Harry becomes obsessed with returning to it, leading to his late-night visits to the mirror room. Dumbledore eventually intervenes, explaining the mirror's powers and the danger of becoming lost in its illusions"
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:57]:
```suggestion
      "summary": "Harry discovers a mysterious mirror in an abandoned classroom at Hogwarts. When he looks into the mirror, he sees himself with his deceased parents, James and Lily Potter, who he never had a chance to know. The mirror shows the deepest desires of a person's heart, and Harry becomes obsessed with returning to it, leading to his late-night visits to the mirror room. Dumbledore eventually intervenes, explaining the mirror's powers and the danger of becoming lost in its illusions."
```

File: backend/db/data/books/1.json [hunk 2]
```diff
@@ -10,71 +10,71 @@
...
+      "summary": "Hagrid wins a dragon egg in a card game and becomes obsessed with hatching it. As the egg is about to hatch, Hagrid asks Harry and Hermione to help him smuggle the dragon, named Norbert, out of the castle to Charlie Weasley, who works with dragons in Romania. The trio's late-night excursion to the tower where Norbert is kept leads to a dangerous encounter with Draco Malfoy. They successfully pass Norbert to Charlie's friends, but they get caught by Mr Filch."
     },
     {
       "title": "The Forbidden Forest",
-      "order": "15"
+      "summary": "Harry and Hermione serve detention in the Forbidden Forest as punishment for being out of their common room after hours. They are accompanied by Hagrid and encounter dangerous creatures, including a wounded unicorn and a mysterious centaur named Firenze. During their time in the forest, they witness a hooded figure drinking the blood of the unicorn. Firenze explains that drinking unicorn blood allows a person to prolong their life, but at a terrible cost – that person becomes cursed. Harry begins to suspect that Snape wants to obtain the Philosopher's Stone to help Voldemort regain his strength."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:69]:
```suggestion
      "summary": "Harry and Hermione serve detention in the Forbidden Forest as punishment for being out of their common room after hours. They are accompanied by Hagrid and encounter dangerous creatures, including a wounded unicorn and a mysterious centaur named Firenze. During their time in the forest, they witness a hooded figure drinking the blood of the unicorn. Firenze explains that drinking unicorn blood allows a person to prolong their life, but at a terrible cost – the drinker becomes cursed. Harry begins to suspect that Snape wants to obtain the Philosopher's Stone to help Voldemort regain his strength."
```

File: backend/db/data/books/1.json [hunk 3]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "Through the Trapdoor",
-      "order": "16"
+      "summary": "Harry, Ron, and Hermione, armed with the knowledge of Fluffy guarding the trapdoor, embark on a perilous journey to reach the Philosopher's Stone before it falls into the wrong hands. These include Devil's Snare, a plant that tries to strangle them, a life-sized wizard's chess match, a game of flying keys, and potions that must be consumed in the correct order to proceed. In the last puzzle it turns out that only Harry can proceed further, and Hermione decides to turn back. Harry encounters an unexpected figure waiting for him."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:73]:
```suggestion
      "summary": "Harry, Ron, and Hermione, armed with the knowledge of Fluffy guarding the trapdoor, embark on a perilous journey to reach the Philosopher's Stone before it falls into the wrong hands. These include Devil's Snare, a plant that tries to strangle them, a life-sized wizard's chess match, a game of flying keys, and potions that must be consumed in the correct order to proceed. In the last puzzle it turns out that only Harry can proceed further, and Hermione decides to turn back to Ron. Harry encounters an unexpected figure waiting for him."
```

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Amazing work! Thank you for all your help. Amazing. Just some smaller nitpicks, see my comments.

File: backend/db/data/books/1.json [hunk 4]
```diff
@@ -10,71 +10,71 @@
   "chapters": [
     {
       "title": "The Boy Who Lived",
-      "summary": ""
+      "summary": "In the first chapter, the author introduces the Dursley family: Mr. and Mrs. Dursley and his son Dudley. They live on Privet Drive number four and are proud of being completely normal. One day, Mr. Dursley notices many strange things: people wearing odd-looking colorful robes and talking about 'You-Know-Who', a peculiar cat and strangers who hug him and tell him to celebrate. That night, Albus Dumbledore, Professor McGonagall (the tabby cat Animagus) and Rubeus Hagrid arrive. They discuss the tragic deaths of James and Lily Potter and the survival of their son Harry. They leave the baby on the Dursleys' doorstep with a letter explaining the circumstances."
```
[the remark this answers is not in the document]
Comment (kat-kan) [reply backend/db/data/books/1.json]: updated

File: backend/db/data/books/1.json [hunk 5]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "The Letter from No One",
-      "summary": ""
+      "summary": "It's a summer holiday before Harry and Dudley go to different schools - Harry's local school and Dudley's expensive private school. One morning during breakfast, the mail arrives at Dursley's home. Unexpectedly, there is a first-ever letter to Harry and on top of that, a strange-looking one. Dursleys don’t allow Harry to read his letter and the next day more letters get into the house by all means, no matter how hard Dursleys try to destroy all of them. Uncle Vernon decides to run away from the problem and takes the whole family to a hotel, but the mail is following them. Finally, they end up in the old cabin on the island which can be accessed only by a boat, hoping they get rid of the letters. Harry is counting the seconds till his 11 birthday and suddenly someone is hammering on the cabin’s door."
```
[the remark this answers is not in the document]
Comment (kat-kan) [reply backend/db/data/books/1.json]: updated with one exception - I googled and "letter to sb" seems to be the correct form :)

**Comment by kat-kan** [comment]
hi @danielschuster-muc thank You for the detailed review! I've updated the file with the suggestions :)

File: backend/db/data/books/1.json [hunk 6]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "The Journey from Platform Nine and Three-Quarters",
-      "summary": ""
+      "summary": "Harry is at King's Cross Station with the Dursleys, preparing to catch the train to Hogwarts. However, they are unable to find Platform Nine and Three-Quarters. In a moment of frustration and desperation, Harry and his new friends Ron Weasley and Hermione Granger, whom he met on the platform, decide to run into the barrier between platforms. They miraculously pass through to the hidden platform, and this unexpected encounter with Ron and Hermione marks the beginning of Harry's magical journey. Additionally, Harry also encounters Draco Malfoy, a fellow student on the Hogwarts Express, who leaves a distinct impression with his aloof and arrogant demeanor."
```
[the remark this answers is not in the document]
Comment (kat-kan) [reply backend/db/data/books/1.json]: Sorry I didn't notice this one before. I proposed a new version of this chapter summary :)

File: backend/db/data/books/1.json [hunk 7]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "The Sorting Hat",
-      "summary": ""
+      "summary": "Harry arrives at Hogwarts School of Witchcraft and Wizardry and is sorted into his house during the Sorting Hat ceremony. The hat briefly considers placing him in Slytherin, but ultimately chooses Gryffindor after Harry expresses a strong desire not to be in Slytherin. Throughout the ceremony, Harry becomes friends with Ron Weasley, who is also sorted into Gryffindor, and is introduced to the concept of the four houses at Hogwarts. Harry also has his first encounters with several of the Hogwarts teachers, including Professor Albus Dumbledore, Professor Severus Snape, Professor Minerva McGonagall, and the quirky Professor Quirrell, who has a turban concealing a secret."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:37]:
```suggestion
      "summary": "Harry arrives at Hogwarts School of Witchcraft and Wizardry and is sorted into his house during the Sorting Hat ceremony. The hat briefly considers placing him in Slytherin, but ultimately chooses Gryffindor after Harry expresses a strong desire not to be in Slytherin. Throughout the ceremony, Harry becomes friends with Ron Weasley, who is also sorted into Gryffindor. Together they have their first encounters with several of the Hogwarts teachers, including Professor Dumbledore, Professor Snape, Professor McGonagall, and the quirky Professor Quirrell, who has a turban concealing a secret."
```
We should probably mention the revealing of Voldemorts turban in the second last chapter.

File: backend/db/data/books/1.json [hunk 8]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "The Potions Master",
-      "summary": ""
+      "summary": "Harry attends his first Potions class at Hogwarts, taught by the enigmatic Professor Severus Snape. He quickly discovers that Snape has a strong dislike for him, leading to a tense and challenging class experience. Harry also experiences other classes, such as Transfiguration taught by Professor McGonagall and Charms with Professor Flitwick. Furthermore, Harry becomes aware of the break-in at Gringotts bank. "
     },
     {
       "title": "The Midnight Duel",
-      "summary": ""
+      "summary": "Harry attends his first Flying lesson, where he gets his first taste of Quidditch, the wizarding sport he quickly becomes passionate about. Harry's remarkable talent on a broomstick is noticed by Professor McGonagall, who recommends him for the Gryffindor Quidditch team. The tension between him and Draco Malfoy steadily escalates. Harry, Ron, and Hermione sneak out of their dormitories at night to challenge Malfoy to a duel. Harry and his friends do not encounter Malfoy, but instead, they stumble upon a massive three-headed dog guarding a mysterious trapdoor."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:45]: Maybe we could have it more like: 
During his first flying lessons, hold by Madam Hooch, Harry's remarkable talent...

Also the connection between this event and the midnight duel is somehow missing.

File: backend/db/data/books/1.json [hunk 9]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "Hallowe'en",
-      "order": "10"
+      "summary": "Hermione Granger impresses her peers with her mastery of the Levitation Charm, 'Wingardium Leviosa' by making a feather float. However, her approach leads to a conflict with Ron, who finds her bossy. Later, during the Halloween feast, a troll is accidentally let into the castle. The troll makes its way into the girls' bathroom, where it threatens Hermione, who is alone and unaware of the danger. Harry and Ron come to her rescue, successfully defeating the troll and solidifying their friendship."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:49]:
```suggestion
      "summary": "Hermione Granger impresses her peers with her mastery of making a feather float with the Levitation Charm 'Wingardium Leviosa'. However, her approach leads to a conflict with Ron, who finds her bossy. Later, during the Halloween feast, a troll is accidentally let into the castle. The troll makes its way into the girls' bathroom, where it threatens Hermione, who is alone and unaware of the danger. Harry and Ron, wondering where Hermione is, come to her rescue, successfully defeating the troll and solidifying their friendship."
```

File: backend/db/data/books/1.json [hunk 10]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "Quidditch",
-      "order": "11"
+      "summary": "Harry joins the Gryffindor Quidditch team as the youngest Seeker in a century. During his first Quidditch match against Slytherin, Harry experiences a series of ups and downs, including Professor Snape's alleged attempt to jinx his broomstick, making it go haywire. Harry's determination prevails and he manages to catch the Golden Snitch, securing the victory for Gryffindor. After the match, Harry, Ron, and Hermione pay a visit to Hagrid to discuss Snape's role in the broomstick incident. Hagrid accidentally reveals information that a three-headed dog is actually his and protects a secret of Dumbledore and Nicholas Flamel."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:53]:
```suggestion
      "summary": "Harry joins the Gryffindor Quidditch team as the youngest Seeker in a century. During his first Quidditch match against Slytherin, Harry experiences a series of ups and downs, including Professor Snape's alleged attempt to jinx his broomstick, making it go haywire. Harry's determination prevails and he manages to catch the Golden Snitch, securing the victory for Gryffindor. After the match, Harry, Ron, and Hermione pay a visit to Hagrid to discuss Snape's role in the broomstick incident. Hagrid accidentally reveals information that it is his three-headed dog and that it actually protects a big secret."
```

File: backend/db/data/books/1.json [hunk 11]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "The Mirror of Erised",
-      "order": "12"
+      "summary": "Harry discovers a mysterious mirror in an abandoned classroom at Hogwarts. When he looks into the mirror, he sees himself with his deceased parents, James and Lily Potter, who he never had a chance to know. The mirror shows the deepest desires of a person's heart, and Harry becomes obsessed with returning to it, leading to his late-night visits to the mirror room. Dumbledore eventually intervenes, explaining the mirror's powers and the danger of becoming lost in its illusions."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:57]:
```suggestion
      "summary": "Harry discovers a mysterious mirror in an abandoned classroom at Hogwarts. When he looks into the mirror, he sees himself with his deceased parents, James and Lily Potter, who he never had a chance to know. Harry quickly becomes obsessed with returning to the mirror, leading to many late-night visits to the mirror room. Professor Dumbledore eventually intervenes, explaining the mirror's powers of showing the deepest desires of a person's heart, and the danger of becoming lost in its illusions. He mentions that he now will move the mirror to another place."
```

File: backend/db/data/books/1.json [hunk 12]
```diff
@@ -10,71 +10,71 @@
...
+      "summary": "Harry and his friends learn about Nicolas Flamel, a renowned alchemist who is believed to have created the Philosopher's Stone, an object with incredible powers. They discover that Flamel's work with the stone has contributed to his extraordinary longevity, and it's also rumored to produce the Elixir of Life, a potion that grants immortality. Harry accidentally overhears a mysterious conversation between Professor Snape and Professor Quirrell. Snape expresses his distrust of Quirrell's loyalty and his urgent need to know how to access the Philosopher's Stone, which is stored at Hogwarts."
     },
     {
       "title": "Norbert the Norwegian Ridgeback",
-      "order": "14"
+      "summary": "Hagrid wins a dragon egg in a card game and becomes obsessed with hatching it. As the egg is about to hatch, Hagrid asks Harry and Hermione to help him smuggle the dragon, named Norbert, out of the castle to Charlie Weasley, who works with dragons in Romania. The trio's late-night excursion to the tower where Norbert is kept leads to a dangerous encounter with Draco Malfoy. They successfully pass Norbert to Charlie's friends, but they get caught by Mr Filch."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:65]: Maybe you start this chapter with: 
During another visit at Hagrid's hut he reveals them... and ask them to help...

Also should mention that Ron is not capable of helping them as he got bit by the dragon, which explains why only Harry and Hermione get punished in the next chapter.

File: backend/db/data/books/1.json [hunk 13]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "Through the Trapdoor",
-      "order": "16"
+      "summary": "Harry, Ron, and Hermione, armed with the knowledge of Fluffy guarding the trapdoor, embark on a perilous journey to reach the Philosopher's Stone before it falls into the wrong hands. These include Devil's Snare, a plant that tries to strangle them, a life-sized wizard's chess match, a game of flying keys, and potions that must be consumed in the correct order to proceed. In the last puzzle it turns out that only Harry can proceed further, and Hermione decides to turn back to Ron. Harry encounters an unexpected figure waiting for him."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:73]:
```suggestion
      "summary": "Harry, Ron, and Hermione, armed with the knowledge of Fluffy guarding the trapdoor, embark on a perilous journey to reach the Philosopher's Stone before it falls into the wrong hands. These include Devil's Snare, a plant that tries to strangle them, a life-sized wizard's chess match, a game of flying keys, and potions that must be consumed in the correct order to proceed. In the last puzzle it turns out that only Harry can proceed further, and Hermione decides to turn back to Ron, who got knocked out earlier in game of chess. Harry encounters an unexpected figure waiting for him."
```

File: backend/db/data/books/1.json [hunk 14]
```diff
@@ -10,71 +10,71 @@
...
     },
     {
       "title": "The Man with Two Faces",
-      "order": "17"
+      "summary": "Harry confronts Professor Quirrell, who is attempting to steal the Stone. Harry faces Voldemort, who is attached to the back of Quirrell's head. Voldemort demands that Harry give him the Philosopher's Stone, but Harry's courage and selflessness protect the Stone from falling into Voldemort's grasp. The Philosopher's Stone is destroyed to prevent it from falling into the wrong hands. Dumbledore explains to Harry that his mother's love and the protection it provided played a crucial role in safeguarding the Stone."
```
Comment (danielschuster-muc) [inline backend/db/data/books/1.json:77]: here it should also reveal that Quirrell actually is "having Voldemort on the back of his head", you know what I mean?

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
To be honest, it's quite hard for me to review this pr. I really want to have good chapter summaries and I am quite perfectionistic in that matter, that's why it is a lot of back and forth. Hope you understand it.

**Comment by kat-kan** [comment]
@danielschuster-muc I'll work on it tomorrow, is there anything I can do to make reviewing changes easier?

**Comment by danielschuster-muc** [comment]
> @danielschuster-muc I'll work on it tomorrow, is there anything I can do to make reviewing changes easier?

That would be amazing, super sorry for all that work. 
It's more like the more I get into it, the more things I find 😂.

File: backend/db/data/books/1.json (hunk 14 above)
Comment (kat-kan) [reply backend/db/data/books/1.json:77]: @danielschuster-muc actually I have in the summary "Harry faces Voldemort, who is attached to the back of Quirrell's head." but maybe I can rework this sentence

**Comment by kat-kan** [comment]
Oops, I guess there's nothing I can do about it :)

File: backend/db/data/books/1.json (hunk 14 above)
Comment (danielschuster-muc) [reply backend/db/data/books/1.json:77]: Oops, overread it, sorry

**Comment by kat-kan** [comment]
@danielschuster-muc fixed, hope this is the last iteration

**Comment by danielschuster-muc** [comment]
> @danielschuster-muc fixed, hope this is the last iteration

Looks good so far, you could just resolve the last few conversations? Besides that I guess it's almost ready 👍

**Comment by kat-kan** [comment]
@danielschuster-muc sorry must've missed some remarks earlier.

**Review by danielschuster-muc — APPROVED** [review]
🥇 It's done I would say!  🎉
Thank you very much for your patience and contribution!

**Comment by kat-kan** [comment]
Thank You @danielschuster-muc :)

**Pull request merged**